main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/gorilla/mux"
+)
+
+//===========================================================================================

```

Comment:
Remove these comments. 
In most cases, a new line is enough to separate different sections of code. If not, you can always create new function and call it.